daemon/container_windows.go

```diff
@@ -38,6 +39,16 @@ func (container *Container) initializeNetworking() error {
 	return nil
 }
 
+// ConnectToNetwork connects a container to the network
+func (container *Container) ConnectToNetwork(idOrName string) error {
```

Comment:
Just making sure here: Windows containers still belong to a default network and have the same connectivity than today? We just don't benefit from "dynamic" network (dis)connections?